src/components/Map/Map.js

```diff
@@ -176,34 +176,36 @@ class Map extends PureComponent {
     }
 
     const queryableLayers = getQueryableLayers("pointermove", layers, map);
-    getFeatureInfoAtCoordinate(coordinate, queryableLayers).then((newInfos) => {
-      // If the featureInfos contains one from a priority layer.
-      // We display only these featureInfos.
-      // See DirektVerbindungen layers for an example.
-      const priorityLayersInfos = newInfos.filter(({ layer }) =>
-        layer.get("priorityFeatureInfo"),
-      );
-      const hasPriorityLayer = !!priorityLayersInfos.length;
-
-      const otherLayersInfos = newInfos.filter(
-        ({ layer }) => !layer.get("priorityFeatureInfo"),
-      );
-
-      const infos = hasPriorityLayer ? priorityLayersInfos : otherLayersInfos;
-
-      // Show the hover style if the layer has the method.
-      infos.forEach(({ layer, features }) => {
-        if (!layer.get("priorityFeatureInfo")) {
-          layer?.hover?.(features);
-        }
-      });
-
-      map.getTarget().style.cursor = infos.find(
-        ({ features }) => !!features.length,
-      )
-        ? "pointer"
-        : "auto";
-    });
+    getFeatureInfoAtCoordinate(coordinate, queryableLayers, "pointermove").then(
```

Comment:
this third parameter seems a bad idea. can we avoid it ?

Reply:
I propose to do the fetchKilometrage call inside the Kilometrage popup. What do you think ?

Reply:
I also thought about that, I liked the idea of having the Popup just handle the UI and letting the layer handle the data. Is there a benefit from doing it in the popup?

And why is the eventType Parameter bad? I thought it might be useful also for other use cases and was surprised we didn't already have a way to access the event. Maybe we could pass in the whole event, not just the type

Reply:
if you want a hell of a mess yes. If you start doing this  you will lose sight on what does the getFeatureInfoAtCoordinates , you will have to check every layer every time to be sure what's happening. 

What you could do is renaming the parameter: 'loadExternalData`  true or false, default to true. and if true, this parameter will append some extenal data to the feature your request when you want it .    

Ideally we should have a hasFeatureInfoAtCoordinate(coordiante, layers) on  layers. And this function will only test if there a clickable feature for this layer at the coordinate. Can you do it  ?